src/regexp.c

```diff
@@ -1205,6 +1231,21 @@ reg_getline(linenr_T lnum)
     return ml_get_buf(rex.reg_buf, rex.reg_firstlnum + lnum, FALSE);
 }
 
+/*
+ * Get length of line "lnum", which is relative to "reg_firstlnum".
+ */
+    static colnr_T
+reg_getlinelen(linenr_T lnum)
```

Comment:
we use `_len` suffix typically, so I think this deserves to be `reg_getline_len()` as well

Reply:
`reg_get_line_len()`?